apps/block_scout_web/assets/js/lib/transaction_history_chart.js

```diff
@@ -79,6 +76,20 @@ class TransactionHistoryChart {
 
   update (transactionHistoryData) {
     this.transaction.data = getTransactionHistoryData(transactionHistoryData)
+
+    const max = Math.max(...this.transaction.data.map(d => d.y)) + 100
+
+    config.options.scales.yAxes[0].ticks = {
+      maxTicksLimit: 4,
```

Comment:
could not see the tick mark on the graph

Reply:
This value is a mistake, is related to grid lines and grid lines are disabled